inference-engine/tests/functional/plugin/shared/src/single_layer_tests/transpose.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <tuple>
+#include <string>
+#include <vector>
+#include <memory>
+#include <functional>
+#include <functional_test_utils/skip_tests_config.hpp>
+
+#include "ie_core.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+
+#include "common_test_utils/common_utils.hpp"
+#include "functional_test_utils/layer_test_utils.hpp"
+
+#include "single_layer_tests/transpose.hpp"
+
+namespace LayerTestsDefinitions {
+
+std::string TransposeLayerTest::getTestCaseName(testing::TestParamInfo<transposeParams> obj) {
+    InferenceEngine::Precision netPrecision;
+    std::vector<size_t> inputShapes, inputOrder;
+    std::string targetDevice;
+    std::tie(inputOrder, netPrecision, inputShapes, targetDevice) = obj.param;
+    std::ostringstream result;
+    result << "IS=" << CommonTestUtils::vec2str(inputShapes) << "_";
+    result << "inputOrder=" << CommonTestUtils::vec2str(inputOrder) << "_";
+    result << "netPRC=" << netPrecision.name() << "_";
+    result << "targetDevice=" << targetDevice;
+    return result.str();
+}
+
+void TransposeLayerTest::SetUp() {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
```

Comment:
Looks like extra line again. Sorry for comment duplication

Reply:
Same as or reduce